modules/internal/localeModels/flat/LocaleModel.chpl

```diff
@@ -31,6 +31,24 @@ module LocaleModel {
   use LocaleModelHelpFlat;
   use LocaleModelHelpMem;
 
+  //
+  // The task layer calls these to convert between full sublocales and
+  // execution sublocales.  Full sublocales may contain more information
+  // in some locale models, but not in this one.
+  //
+  export
+  proc chpl_localeModel_sublocToExecutionSubloc(full_subloc:chpl_sublocID_t)
```

Comment:
`chpl_localeModel_sublocToExecutionSubloc` is pretty long name. I'd prefer something shorter too.